### DISCUSSION TITLE
Could not reset password

### DISCUSSION THREAD

**Discussion by tamy-le** [community/community#119463] [Projects and Issues] [answered]
### Select Topic Area

Bug

### Feature Area

Issues

### Body

1.

The message is ambiguous, Maybe after hitting "send password reset email" it should have said:

Checked mail!!!

![image](https://github.com/community/community/assets/157474751/60a85721-602e-431c-bc9b-4db55e0b3808)

2. I could not receive mail for reset password even after 4 hours and more.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by tamy-le** [comment, marked as answer]
After 12 hours, github sent me password

**Reply by DGJADEJA** [reply]
dhamandrasinh63@gmail.com

**Comment by coolspot18** [comment]
Same, took like 6 or 10 hours, why is the password reset so slow?

**Comment by DGJADEJA** [comment]
My gmail account password forgot
Gmail (dhamandrasinh63@gmail.com)

**Comment by dhurka-a-m-db** [comment]
The copied user, silpa-r.padhy@db.com, i sfacing issues in resetting the password. Could you please help.

**Comment by ghost** [comment]
My gmail account password forgot
gmail(harshad2043@gmail.com)
